docs/how-to/enforce-permissions/user-permissions.mdx

```diff
@@ -160,7 +160,40 @@ UserPermissions permissions = permit.getUserPermissions(
 );
 
 ```
-</TabItem>
 
+##  Get user permissions directly from opa
+
+:::info Info
+Not Support FactDB
+:::
+
+```java
+import io.permit.sdk.Permit;
+import io.permit.sdk.PermitConfig;
+import io.permit.sdk.enforcement.*;
+import io.permit.sdk.util.Context;
+import java.util.Arrays;
+
+
+Permit permit = new Permit(
+    new PermitConfig.Builder("[YOUR_API_KEY]").build()
+);
+
+Context context = new Context();
+context.put("enable_abac_user_permissions", new Boolean(true));
+
+UserPermissions permissions = permit.getUserPermissionsFromOPA(
+	new GetUserPermissionsQuery(
+		User.fromString("john@doe.com"), // user key
+		null, // tenants filter is not required for ABAC
+		Arrays.asList("document", "__tenant"), // resource types is always required for ABAC, __tenants is required to not ignore RBAC-based permissions
+		null, // resources not required
+		context,
```

Comment:
this makes your example harder to understand ( I guess because you copied it from the ABAC example ). 
copy the most simple example here to demonstrate only the relevant function